test/Base_Tests/src/Semantic/Arrow_Interop_Spec.enso

```diff
@@ -0,0 +1,41 @@
+from Standard.Base import all
+import Standard.Base.Errors.Common.Index_Out_Of_Bounds
+import Standard.Base.Errors.Common.Unsupported_Argument_Types
+import Standard.Base.Data.Arrow_Array_Builder.Arrow_Array_Builder
+from Standard.Test import all
+
+foreign arrow new_arrow = """
+    new[Int8]
+
+## Used only for testing immutability
+Vector.set self at item = set_builtin self at item
```

Comment:
I'd prefer this to not be an extension function. Why not call `set_builtin` directly?

By being extension function it is unclear where we call the `Vector.set` method and when we call the method on another type. With direct call it would be more explicit.

Reply:
> Is this a left over? 

yes